wordpress-mobile/WordPress-iOS
# Splitview for Reader

Implements `UISplitViewController` for the Reader section of the app.

The left pane always shows the list of streams and the right pane shows stream contents, post details, comments, etc. This should behave the same as Calypso's Reader.

To test:

* Check everything works as you'd expect, including with multitasking and on iPhone.
* When you add a new tag, it should become selected in the list and get scrolled into view. If you're on an iPad and the splitview is expanded then you should be shown the content of the tag. If the splitview is collapsed or you're on iPhone, then it'll just be highlighted and then deselected.
* Check things get reset when signing out of the app and into a different account.
* State restoration should work!

This PR doesn't include fullscreen for reader post details – that's coming in a separate PR (next week).

Needs review: @kurzee 

@aerych Would you mind double checking that things look as you'd expect too?
